easybuild/easyconfigs/l/LZO/LZO-2.06-goolf-1.4.10.eb

```diff
@@ -15,18 +15,21 @@ version = '2.06'
 homepage = 'http://www.oberhumer.com/opensource/lzo/'
 description = "LZO-2.06: Portable lossless data compression library"
 
-sources = [SOURCELOWER_TAR_GZ]
 source_urls = [homepage + 'download/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ff79e6f836d62d3f86ef6ce893ed65d07e638ef4d3cb952963471b4234d43e73']
 
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 
-sanity_check_paths = {
-    'files': [],
-    'dirs': ['lib', 'include']
-}
+configopts = '--enable-shared'
+
+parallel = 1    # this is a very conservative choice
 
 runtest = 'test'
 
-parallel = 1    # this is a very conservative choice
+sanity_check_paths = {
+    'files': ['lib/liblzo2.a', 'lib/liblzo2.so'],
```

Comment:
@pescobar Overlooked this earlier, we should use `SHLIB_EXT` rather than hardcode `.so`

Fixed in https://github.com/pescobar/easybuild-easyconfigs/pull/68